src/graphql/historicalProposal.utils.ts

```diff
@@ -0,0 +1,32 @@
+import { FragmentType, getFragmentData } from 'gql'
```

Comment:
we could rename this to historical-proposal.utils.ts so it follows the existing file naming pattern

Reply:
f00bec5